docs/docs/keys/developers.md

```diff
@@ -36,9 +36,9 @@ A good choice for a remote signer is [Web3signer](https://docs.web3signer.consen
 
 ## Key Management Recommendation for Developers
 
-The Node can implement a feasible and sufficient method of loading the keys. This is asking for a path to a keystore folder. This keystore needs to follow some structure either the Service knows how to read, or an AVS [plugin](../plugin/intro.mdx) knows how to transform to something the Service knows how to read.
+The Node can implement a feasible and sufficient method of loading the keys. This is asking for a path to a keystore folder. This keystore needs to follow some structure either the Service knows how to read, or an AVS [plugin](/docs/category/plugin) knows how to transform to something the Service knows how to read.
```

Comment:
In this paragraph, are **Node** and **Service** the same thing? If yes, I propose use the same term in both cases. Also, I think we need to include the **AVS** term: **AVS Node** or **AVS Service**.

Reply:
Done [b781a58](https://github.com/NethermindEth/el-node-spec/pull/54/commits/b781a58a87e5e3d868124c43497aa0122840fee6)